src/styles/blocks/form-field--big.css

```diff
@@ -0,0 +1,24 @@
+.form-field__first {
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  background-image: url(/src/images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  padding-left: 62px;
+  height: 69px;
+  width: 100vw;
```

Comment:
Using `width: 100vw` can cause horizontal scrolling issues on smaller screens due to the scrollbar width. Consider using `width: 100%` instead to ensure the input field fits within its container.